### ISSUE TITLE
2.0 Changelog

### ISSUE THREAD

**Issue by SteveEdson** [phpseclib/phpseclib#842]
Is there a changelog available for 2.0 since it has now been merged into master? Also, the website documentation seems to reference 1.0 still.

Thanks

**Comment by terrafrost** [comment]
> Is there a changelog available for 2.0 since it has now been merged into master?

No. In theory, most 2.0 changes will be merged into master and most 1.0 changes merged into 2.0.

> Also, the website documentation seems to reference 1.0 still.

That's correct. 1.0 is more of a LTS version whereas 2.0 is not. Making the website document both 1.0 and 2.0 would be a lot of work and altho it'd be good, it is, to be perfectly honest, not among my highest of my priorities. The 2.0 API is basically the 1.0 API namespaced. Also, it requires an autoloader. Those are about the only changes.

3.0 will feature some pretty drastic API changes so updating the website for that one will be a much higher priority. But if things go as planned 3.0 will probably be LTS like 1.0.

In the mean time if you want to have phpseclib.sourceforge.org document both 1.0 and 2.0 it'd be good to submit a PR to that project:

https://github.com/phpseclib/docs

**Comment by mnapoli** [comment]
Please see #912 It is based on your comment:

> The 2.0 API is basically the 1.0 API namespaced. Also, it requires an autoloader. Those are about the only changes.
